src/model/chart-model.ts

```diff
@@ -37,10 +37,21 @@ export interface HandleScrollOptions {
 export interface HandleScaleOptions {
 	mouseWheel: boolean;
 	pinch: boolean;
-	axisPressedMouseMove: boolean;
+	axisPressedMouseMove: AxisPressedMouseMoveOptions;
```

Comment:
```suggestion
	axisPressedMouseMove: AxisPressedMouseMoveOptions | boolean;
```


Shouldn't be like this? Otherwise there is no difference between `HandleScaleOptionsInternal` and `HandleScaleOptions` though.

Reply:
Ok, I'll fix it after merge, don't worry.